src/cart/crt_init.c

```diff
@@ -174,6 +175,9 @@ prov_data_init(struct crt_prov_gdata *prov_data, crt_provider_t provider,
 	prov_data->cpg_max_unexp_size = max_unexpect_size;
 	prov_data->cpg_primary = primary;
 
+	for (i = 0; i < CRT_SRV_CONTEXT_NUM; i++)
+		prov_data->cpg_used_idx[i] = false;
```

Comment:
is that necessary? don't/can't you initialize that struct to {0} somewhere before you call this init?

Reply:
we can, but i like explicit init